- RoadObject: added `isUrban` flag; - EHorizonEdgeMetadata: added `isUrban` flag. (#6178)
mapbox/mapbox-navigation-android: CHANGELOG.md

## Unreleased
#### Features

> - Added `isUrban` flag to `EHorizonEdgeMetadata` where edge is in urban area or not. [#6178](https://github.com/mapbox/mapbox-navigation-android/pull/6178)

Reviewer: maybe?
```suggestion
- Added an `isUrban` flag to `EHorizonEdgeMetadata` which describes if an edge is in an urban area. [#6178](https://github.com/mapbox/mapbox-navigation-android/pull/6178)
```

Author: fixed

> - Added `isUrban` flag to `RoadObject` where object is in urban area or not. Might be `null` if it's not possible to define. [#6178](https://github.com/mapbox/mapbox-navigation-android/pull/6178)

Reviewer: maybe?
```suggestion
- Added an `isUrban` flag to `RoadObject` which describes if an object is in an urban area. [#6178](https://github.com/mapbox/mapbox-navigation-android/pull/6178)
```

Author: fixed
